test_framework/scripts/download-support-bundle.sh

```diff
@@ -35,7 +37,8 @@ SUPPORT_BUNDLE_URL=`kubectl exec -n longhorn-system svc/longhorn-frontend -- bas
 
 SUPPORT_BUNDLE_READY=false
 while [[ ${SUPPORT_BUNDLE_READY} == false ]]; do
-    PERCENT=`kubectl exec -n longhorn-system svc/longhorn-frontend -- curl -H 'Accept: application/json' ${SUPPORT_BUNDLE_URL} | jq -r '.progressPercentage'`
+    RES=`kubectl exec -n longhorn-system svc/longhorn-frontend -- curl -H 'Accept: application/json' ${SUPPORT_BUNDLE_URL}`
+    PERCENT=`echo "${RES}" | jq -r '.progressPercentage'`
```

Comment:
@yangchiu could you explain a bit why the previous pipe will probably not work?

Reply:
`set -e` only works when there is an error in the last command of a pipe.
In our case, the corrupt environment can't even get the longhorn-frontend url, so the error would happen in the first command of the pipe (`kubectl exec`) but `set -e` would not notice this error because of the pipe.
Separating commands instead of using pipe can fix this, but I find `set -o pipefail` is more graceful. Modified this PR to use `set -o pipefail`

Reply:
I see, but my question is why this change will fix the issue of downloading bundle forever? Previously, the script did not have `set -e`.

Reply:
Yes, previously the script didn't have `set -e ` so if there is an error it would stuck in while loop forever.
Add `set -e` won't fix the problem because the error happened in the pipe.
Add `set -eo pipefail` fix the problem because `set -o pipefail` would let the whole pipe fail if one of command failed in the pipe.

Reply:
Ah u are right!